orttraining/orttraining/eager/ort_aten.cpp

```diff
@@ -492,11 +492,11 @@ void resize_impl_ort_(
     // Just resize existing tensor and return
 
     OrtValue new_ort_value = reshape_invoke(
-              invoker,
-              self_ort_value,
-              size,
-              // invoke reshape kernel inplace
-              true);
+        invoker,
+        self_ort_value,
+        size,
+        // invoke reshape kernel inplace
```

Comment:
nit: while maybe not in the formatter, do we have any guidance on putting these above the relevant argument vs inline with it?  (The function below is inlined)

Reply:
I think I have seen both patterns used.... From Microsoft C# guidelines, I tend to be trained to put all arguments on new line... but I don't feel strongly.

According to the Google style guidelines (which I think out clang-format style is based off of), they include both options: https://google.github.io/styleguide/cppguide.html#Function_Calls

Reply:
this was specifically about the comment being on its own line

Reply:
@msftlincoln so your saying move the comment after the true?

Reply:
we can change that in a future PR too, not wanting to restart all the check if this is the only outstanding thing.

Reply:
Thanks for the clarification - there are a few places in this file with this pattern - let's address in follow up.